### PR TITLE
Updated various package dependencies

### PR DESCRIPTION
**Pull request by jkaster**
- fixed failing codegen unit tests

### CODE REVIEW

**Review by josephaxisa — COMMENTED** [review]
Builds fine and all codegen tests pass on my end as well. We are getting some peer dependency warnings at build time though:
```
[4/5] Linking dependencies...
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/api-explorer > @testing-library/user-event@12.1.8" has unmet peer dependency "@testing-library/dom@>=7.21.4".
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/api-explorer > react-test-renderer@17.0.1" has incorrect peer dependency "react@17.0.1".
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/extension-api-explorer > @looker/extension-sdk-react@0.6.5" has unmet peer dependency "@types/react@^16.8".
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/extension-api-explorer > @looker/extension-sdk-react@0.6.5" has unmet peer dependency "@types/react-dom@^16.8".
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/run-it > react-google-charts@3.0.15" has unmet peer dependency "prop-types@>=15".
warning "workspace-aggregator-33d1db65-822b-4a2c-beee-3ddc1d0408da > @looker/run-it > react-google-charts > react-load-script@0.0.6" has unmet peer dependency "prop-types@>=15".
```

File: packages/sdk-codegen/src/sdkModels.spec.ts [hunk 1]
```diff
@@ -893,7 +893,7 @@ describe('sdkModels', () => {
         const actual = apiTestModel.search('dashboard', modelAndTypeNames)
         const methods = allMethods(actual.tags)
         expect(Object.entries(methods).length).toEqual(33)
-        expect(Object.entries(actual.types).length).toEqual(27)
+        expect(Object.entries(actual.types).length).toEqual(29)
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-codegen/src/sdkModels.spec.ts:896]: Yeah, I learned about that long after these tests were written. Would be good to refactor those on another PR. I was just trying to get the critical tests to pass again. And I also want to change these tests to do more than check length (which is very fragile) but instead inspect the results of the matches to make sure they meet the criteria specified.

[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-codegen/src/sdkModels.spec.ts:896]: FWIW, peer dependencies warnings are better than they were before this work, but I suppose they should be fixed as part of this update too. Wish workspaces had better tooling/support for these updates but it's not _that_ painful.

[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-codegen/src/sdkModels.spec.ts:896]: Trying to resolve these dependencies led to all kinds of whack-a-mole scenarios. Those will have to wait for another PR, possibly when everyone else has updated to react 17

[the remark this answers is not in the document]
Comment (josephaxisa) [reply packages/sdk-codegen/src/sdkModels.spec.ts:896]: Thanks for looking into it

**Review by josephaxisa — APPROVED** [review]
👍

**Pull request merged**